MapboxNavigation/NavigationMapView.swift

```diff
@@ -87,6 +87,8 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
     var animatesUserLocation: Bool = false
     var altitude: CLLocationDistance = defaultAltitude
     var routes: [Route]?
+    var altRouteSourceIdentifiers: [String] = []
+    var altRouteLayerIdentifiers: [String] = []
```

Comment:
We only need one layer for all alternative routes. In fact, we might even be able to combine that layer into the one for the primary route: the expression for `lineColor` would get more complex, but there would be a net decrease in code I think.

Reply:
Sounds good. Selecting routes will still be doable?

Reply:
Yes, feature querying should get you the specific feature within the overall shape collection feature that was selected. Then you can use the feature’s identifier to distinguish one route from another.